.claude/skills/jira-triage/SKILL.md

```
@@ -0,0 +1,1300 @@
+    "RHOAIENG-12345": "Issue title for readability",
+    "RHOAIENG-12346": "Another issue in the query result"
+  },
+  "operations": [
+    {
+      "issueKey": "RHOAIENG-12345",
+      "action": "ADD_LABEL",
+      "params": { "labels": ["needs-info"] },
+      "reason": "No reproduction steps, reporter inactive for 30+ days"
+    },
+    {
+      "issueKey": "RHOAIENG-12346",
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)